docs/4.0/getting-started/theming.md

```diff
@@ -76,6 +76,8 @@ With that setup in place, you can begin to modify any of the Sass variables and
 
 Every Sass variable in Bootstrap 4 includes the `!default` flag allowing you to override the variable's default value in your own Sass without modifying Bootstrap's source code. Copy and paste variables as needed, modify their values, and remove the `!default` flag. If a variable has already been assigned, then it won't be re-assigned by the default values in Bootstrap.
 
+You will find the complete list of variables that can be overriden and their default values in our `scss/_variables.scss` file.
```

Comment:
Hi @pgiraud thanks for the contribution, I this is makes it more clear. I'd change the wording from "our" to "Bootstrap's". For example:
"You will find the complete list of Bootstrap's variables and their default values in "

Reply:
I agree with you. I don't like the 'our' either. I just tried to be consistent.
Changed.